docs/reference/aggregations/metrics/max-aggregation.asciidoc

```diff
@@ -67,3 +67,24 @@ Let's say that the prices of the documents in our index are in USD, but we would
 }
 --------------------------------------------------
 
+==== Missing value
+
+The `missing` parameter defines how documents that miss a value should be treated.
+By default they will be ignored but it is also possible to treat them as if they
+had a value.
+
+[source,js]
+--------------------------------------------------
+{
+    "aggs" : {
+        "grade_max" : {
+            "max" : {
+                "field" : "grade",
+                "missing": 10 <1>
+            }
+        }
+    }
+}
+--------------------------------------------------
+
+<1> Document that don't have a value for the `grade` field will be treated as if they had `10` as a value
```

Comment:
Documents without a value in the `quantity` field will fall into the same bucket as documents that have the value `10`.